docs/UserGuide.adoc

```diff
@@ -133,6 +135,7 @@ Format: `deldeck INDEX_OF_DECK`
 ****
 * Deletes the deck at the specified `INDEX_OF_DECK`.
 * `INDEX_OF_DECK` must be a positive integer from 1 onwards and is based on the currently displayed list.
+* This operation is disabled when user navigates into a deck.
```

Comment:
Looks good! I think preventing the user from deleting a deck when he/she is inside a deck will as we have less cases to handle.